### ISSUE TITLE
jwt-authentication example

### ISSUE THREAD

**Issue by johnrackles** [vercel/examples#33]
When I try to run the example locally, I get:

```
error - node_modules/jose/dist/browser/jws/flattened/sign.js (11:0) @ new FlattenedSign
TypeError: payload must be an instance of Uint8Array
null
```

Oddly enough, when I deploy it everything seems to work fine. Any Ideas how that happens? I checked, both vercel and my local machine are on node 14.18.1

**Comment by panva** [comment]
This is due to the `vm` module isolation, the [Indexed Collections](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#indexed_collections) are not shared between where libraries are required/imported and the execution context.

I've opened an issue about it https://github.com/vercel/next.js/issues/30477

**Comment by johnrackles** [comment]
For anyone stumbling upon this, it's fixed in v12.0.4-canary.4
